### PR TITLE
Reorder global load instructions of dtva custom kernel

### PR DESCRIPTION
**Pull request by briannwu**
Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA

### CODE REVIEW

File: tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s [hunk 1]
```diff
@@ -65,6 +65,8 @@ custom.config:
    1LDSBuffer: 1
    ScheduleIterAlg: 3
    DepthU: 64
+   StaggerU: 8
+   WorkGroupMapping: 8
```
Comment (hcman2) [inline tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s]: are these 2 parameters can be removed?

**Comment by hcman2** [comment]
Will you add a test yaml to run this kernel?

File: tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s [hunk 2]
```diff
@@ -65,6 +65,8 @@ custom.config:
    1LDSBuffer: 1
    ScheduleIterAlg: 3
    DepthU: 64
+   StaggerU: 8
```
Comment (Jinp800125) [inline tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s]: Is this kernel only supports STU8 and WGM8?

**Review by aazz44ss — APPROVED** [review]

File: tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s (hunk 1 above)
Comment (briannwu) [reply tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s]: done

File: tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s (hunk 2 above)
Comment (briannwu) [reply tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s]: removed

**Comment by briannwu** [comment]
> Will you add a test yaml to run this kernel?
added

File: tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        - LocalWritePerMfma: [-1]
+        - StoreRemapVectorWidth: [0]
+        - StoreVectorWidth: [-1]
+        - SourceSwap: [1]
+        - NumElementsPerBatchStore: [16]
+        - ClusterLocalRead: [1]
+      BenchmarkJoinParameters:
+      BenchmarkFinalParameters:
+        - ProblemSizes:
+          - Exact: [256, 256, 1, 256]
```
Comment (hcman2) [inline tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml:88]: does this kernel support tail and edge? If yes, you may add some edge/tail sizes.

Comment (briannwu) [reply tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml:88]: done

**Review by Jinp800125 — APPROVED** [review]

**Pull request merged**